docs/source/kiva/compiled_path.rst

```diff
@@ -0,0 +1,25 @@
+CompiledPath
+============
+
+A path is a collection of geometry that can be drawn in a graphics context with
+coloring and an affine transformation applied to it. It is the basic unit of
+drawing in a graphics context.
+
+Every graphics context instance has a current path which can be manipulated by
+the :ref:`kiva_path_functions`. However, some drawing operations are easier to
+implement with an independent path instance.
```

Comment:
Maybe add an example or a "why" sentence here.  Ie be explicit about what would make a drawing operation be easier with an independent path instance.  For example like wanting to draw the same shape repeatedly.

Reply:
Yup. Can add a ref to `draw_path_at_points` here